tests/test_pusher.py

```diff
@@ -257,3 +257,13 @@ async def test_push_delete(self):
         self.assertEqual("/metrics/job/my-job", self.server.test_results["path"])
         self.assertEqual("DELETE", self.server.test_results["method"])
         self.assertEqual(valid_result, self.server.test_results["body"])
+
+    async def test_push_delete(self):
+        job_name = "my-job"
+        p = Pusher(job_name, "http://localhost:2022")
```

Comment:
I'm sure there is a better way to do this, but this should represents an endpoint that will reject the request. A more robust test would have an endpoint that just takes longer than a second to reply.